src/video/SDL_egl.c

```diff
@@ -816,6 +816,10 @@ static int SDL_EGL_PrivateChooseConfig(SDL_VideoDevice *_this, SDL_bool set_conf
         attribs[i++] = _this->egl_data->egl_surfacetype;
     }
 
+    if (_this->gl_config.swap_behavior_preserved_bit) {
```

Comment:
This bit is only available if the EGL version >= 1.4, so it should be guarded by an #ifdef, and the current version should be checked before enabling it.

Reply:
Thanks